PKGBUILD

```diff
@@ -0,0 +1,22 @@
+# Maintainer: Your Name <youremail@domain.com>
+pkgname="particle-life"
+pkgrel=1
+pkgver=1.1.3 # todo: make automatic
+pkgdesc="GUI for Particle Life, a particle system showing life-like behaviour"
+arch=("x86_64")
+url="https://github.com/tom-mohr/particle-life-app"
+license=('GPL')
+depends=('java-environment')
+makedepends=('git' 'java-environment>=11')
```

Comment:
The Particle Life App requires Java >= 16, so this here should say `java-environment>=16`, I assume?

Reply:
yep: that it should be